Packs/CommonScripts/Scripts/PrintRaw/CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+## [Unreleased]
```

Comment:
It's added by 'demisto-sdk split-yml'.  Past PR's have required me to add it, so why am I now being asked to remove it?

Reply:
You are right and we have an open issue to remove it. But since the marketplace release, changes need to appear in release notes as opposed to changelog.
You can run `demisto-sdk update-release-notes -p CommonScripts -u revision` to create release notes instead.
I am sorry for the inconvenience.

Reply:
No worries, Shahaf.  Will do.

Reply:
I'll go ahead and remove the changelog, then.

Reply:
😃